client/src/components/questions/QuestionList.jsx

```diff
@@ -73,13 +75,15 @@ const QuestionList = ({ questions, productName, productID }) => {
   return (
     <div>
       <StyledQuestionList>
-        {list.map((q) => (
-          <Question questionData={q} key={q.question_id} productName={productName}/>
-        ))}
+        {list.map((q) => {
+          count++;
```

Comment:
clever! was this how you counted the number of questions/answers? with the count variable? If so, consider labeling the variable 'countingQuestionsForTesting' or something so that other people would easily be able to identify it as NOT critical for running the page

Reply:
good call I will look into refactoring this out at this point, I do not think it is necessary anymore after only rendering dummy data.